sequencer/config/scripts/config.py

```diff
@@ -0,0 +1,84 @@
+from json import dump, load
+
+class Key:
+    def __init__(self, name, secret):
+        self.name = name
+        self.secret = secret
+
+    @classmethod
+    def from_file(cls, filename):
+        assert isinstance(filename, str)
+        with open(filename, 'r') as f:
+            data = load(f)
+        return cls(data['name'], data['secret'])
+
+
+class Committee:
+    def __init__(self, names, consensus_addr, transactions_addr, mempool_addr):
+        inputs = [names, consensus_addr, transactions_addr, mempool_addr]
+        assert all(isinstance(x, list) for x in inputs)
+        assert all(isinstance(x, str) for y in inputs for x in y)
+        assert len({len(x) for x in inputs}) == 1
+
+        self.names = names
+        self.consensus = consensus_addr
+        self.front = transactions_addr
+        self.mempool = mempool_addr
+
+        self.json = {
+            'consensus': self._build_consensus(),
```

Comment:
Is the "execution" field added somewhere else or is it ignored? (Guessing it's somewhere else since I tested these changes and everything works)

Reply:
For now, the parameters.json file is commited ([sequencer/config/parameters.json](https://github.com/lambdaclass/starknet_stack/blob/main/sequencer/config/parameters.json)) and not generated with these scripts; this code is to generate the committee.json which [is configured with the node's `--committee` flag
](https://github.com/lambdaclass/starknet_stack/blob/main/docker-compose.yml#L21), you can find it locally (after running `make run-local` on `sequencer/config/committee.json`